bazel/markdown_to_html/src/helpers.ts

```diff
@@ -0,0 +1,17 @@
+/*!
```

Comment:
Can we avoid files like `helpers.ts`, `utils.ts`, etc? 

For these functions in particular, they're both related to url / links, so maybe a name related to that (`urls.ts`)?

Reply:
How would you feel about a cleanup at the end to expand out the helper files? I don't feel that I can predict how the split will make sense at this point of the migration.